spec/rundeck/client/token_spec.rb

```diff
@@ -70,24 +74,29 @@
   end
 
   describe '.create_token' do
-    context 'when the user exists', vcr: { cassette_name: 'create_token' } do
+    context 'when the user exists',
+            vcr: { cassette_name: 'token/token_create' } do
       before do
-        @token = Rundeck.create_token('admin')
+        @token = Rundeck.create_token('dev')
       end
       subject { @token }
 
       it_behaves_like 'a token'
 
       it 'expects a post to have been made' do
-        expect(a_post('/tokens/admin')).to have_been_made
+        expect(a_post('/tokens/dev')).to have_been_made
       end
     end
   end
 
   describe '.delete_token' do
-    context 'when a token exists', vcr: { cassette_name: 'delete_token' } do
+    context 'when a token exists',
+            vcr: { cassette_name: 'token/token_delete' } do
       before do
-        @token = Rundeck.delete_token('cmJQYoy9EAsSd0905yNjKDNGs0ESIwEd')
+        VCR.use_cassette('token/tokens_user') do
+          @dev_tokens = Rundeck.tokens('dev')
+        end
+        @token = Rundeck.delete_token(@dev_tokens.token.first.id)
```

Comment:
I'm not thrilled with this implementation... May revisit this before finishing up. Tokens specs in general are meh.